parser.go

```diff
@@ -33,9 +33,20 @@ type Entry struct {
 type Option struct {
 	Pos lexer.Position
 
-	Name  string  `( "(" @("."? Ident { "." Ident }) ")" | @("."? Ident { "." Ident }) )`
-	Attr  *string `[ @("."? Ident { "." Ident }) ]`
-	Value *Value  `"=" @@`
+	Name  string     `( "(" @("."? Ident { "." Ident }) ")" | @("."? Ident { "." Ident }) )`
+	Attr  *string    `[ @("."? Ident { "." Ident }) ]`
+	Text  *ProtoText `"=" ( @@`
```

Comment:
Should this just be part of `Value`?

Reply:
Hmm, how would the differentation here work if it was?
```
type ProtoTextField struct {
	Name  string     `@Ident `
	Text  *ProtoText `( ( ":"? @@ )`
	Value *Value     `  | ( ":" @@ ) )`
}
```

Reply:
i.e, `Value` requires `:` before it, while prototext does not necessarily need it

Reply:
Actually, I might see if I can refactor this a bit, potentially parsing values without `:` even though `protoc` fails for these

Reply:
Done, now the parser is slightly more permissive than `protoc` but I think that is a fine tradeoff for a nicer interface